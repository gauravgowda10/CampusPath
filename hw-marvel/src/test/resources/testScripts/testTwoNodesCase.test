# build graph
LoadGraph g TwoNodes.tsv

# check graph is built correctly
ListNodes g
ListChildren g IronMan
ListChildren g SpiderMan

# check BFS
FindPath g IronMan IronMan
FindPath g IronMan SpiderMan
FindPath g SpiderMan IronMan