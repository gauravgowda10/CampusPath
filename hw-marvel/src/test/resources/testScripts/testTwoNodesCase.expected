# build graph
loaded graph g

# check graph is built correctly
g contains: IronMan SpiderMan
the children of IronMan in g are: SpiderMan(Avengers)
the children of SpiderMan in g are: IronMan(Avengers)

# check BFS
path from IronMan to IronMan:
path from IronMan to SpiderMan:
IronMan to SpiderMan via Avengers
path from SpiderMan to IronMan:
SpiderMan to IronMan via Avengers